src/actions/tests/ui.js

```diff
@@ -37,22 +37,18 @@ describe("ui", () => {
 
   it("should update search results", () => {
     const { dispatch, getState } = createStore();
-    expect(getSearchResults(getState())).toEqual({ index: -1, count: 0 });
+    expect(getSearchResults(getState())).toEqual({
+      matches: [],
+      matchIndex: -1,
+      index: -1,
+      count: 0
+    });
 
     const results = { count: 3, index: 2 };
     dispatch(actions.updateSearchResults(results));
     expect(getSearchResults(getState())).toEqual(results);
   });
 
-  fit("should update symbol search results", () => {
-    const { dispatch, getState } = createStore();
-    expect(getSymbolSearchResults(getState())).toEqual([]);
-
-    const results = [{ foo: "foo" }];
-    dispatch(actions.updateSymbolSearchResults(results));
-    expect(getSymbolSearchResults(getState())).toEqual(results);
-  });
```

Comment:
looks like we weren't testing the other tests because of the fit.

We should lint for this.